docs/learn/learn-staking-miner.md

```diff
@@ -48,7 +51,28 @@ order to submit solutions to the NPoS election. When the correct time comes, it
 solution and submits it to the chain. The default miner algorithm is sequential Phragmén with a
 configurable number of balancing iterations that improve the score.
 
-![NPoS election optimization](../assets/staking-miner/NPoS-election-optimization.png)
+## Deposit and reward mechanics
+
+The staking miners are required to pay a deposit in order to post their solutions. The deposit is a
+sum of the `SignedDepositBase`, which is a fixed amount and `SignedDepositByte`, a variable amount
+per KB of solution data. All good solutions are subject to recieving a `SignedRewardBase`.
```

Comment:
fixed typo of 'receiving' and a bit of rephrasing. Also, the deposit collected seems to include a per-weight deposit as well https://github.com/paritytech/substrate/blob/49734dd1d72a00b9d3b87ba397661a63e0e17af3/frame/election-provider-multi-phase/src/signed.rs#L501

```suggestion
The staking miners are required to pay a deposit to post their solutions. This deposit is a
sum of the fixed `SignedDepositBase` amount and the variable`SignedDepositByte` amount, which depends on the size of the solution data. All good solutions are subject to receiving a `SignedRewardBase`.
```